src/EnergyPlus/CMakeLists.txt

```diff
@@ -928,23 +928,22 @@ if(LINK_WITH_PYTHON)
   add_custom_command(
     TARGET energyplusapi
     POST_BUILD # TODO: I don't think we want to quote the generator expression
-               # here
     COMMAND ${Python_EXECUTABLE} "${PROJECT_SOURCE_DIR}/cmake/PythonCopyStandardLib.py" "$<TARGET_FILE:energyplusapi>" "python_standard_lib")
 endif()
 
 if(BUILD_PACKAGE)
   # if we are building package, we need to drop in some API/Plugin stuff
   if(LINK_WITH_PYTHON)
-    # we'll want to grab the standard lib for python plugins TODO: I don't think
-    # we want to quote the generator expression
+    # we'll want to grab the standard lib for python plugins
+    # TODO: I don't think we want to quote the generator expression
     install(DIRECTORY "$<TARGET_FILE_DIR:energyplus>/python_standard_lib/" DESTINATION "./python_standard_lib")
   endif()
   # we'll want to always provide the C API headers
   foreach(HEADER ${API_HEADERS})
     install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/${HEADER}" DESTINATION "./include/EnergyPlus/api")
   endforeach()
-  # we'll want to always grab the python API wrapper TODO: I don't think we want
-  # to quote the generator expression
+  # we'll want to always grab the python API wrapper
+  # TODO: I don't think we want to quote the generator expression
```

Comment:
Unrelated but I think we can remove the TODO. https://cmake.org/pipermail/cmake/2014-June/057768.html

Reply:
```suggestion
```

Reply:
I want to believe it.  But one time about a year ago I was dealing with a strange issue where the generator expression wasn't being evaluated in time and a path was not constructed properly.  It was some interdependency between the generator expression, quotes, and it being used as an argument to a separate CMake file or something.  With fresh eyes and an extra year of experience dealing with this, I would be happy to test this out further.  I don't think I want to add that tweak to this PR though.